{"uid":"7dbcdd63594319d0","name":"Exception test","fullName":"tests.forgot_password_test.TestForgotPassword#test_expected_exception_on_page_title","historyId":"288cd0030c92ba34b4ff4b58ed8ec476","time":{"start":1733708996672,"stop":1733709006934,"duration":10262},"description":"Exception catching","descriptionHtml":"<p>Exception catching</p>\n","status":"broken","statusMessage":"selenium.common.exceptions.TimeoutException: Message: \nStacktrace:\n#0 0x55bfa6a0531a <unknown>\n#1 0x55bfa651b6e0 <unknown>\n#2 0x55bfa656a3e6 <unknown>\n#3 0x55bfa656a681 <unknown>\n#4 0x55bfa65afb04 <unknown>\n#5 0x55bfa658e48d <unknown>\n#6 0x55bfa65aced7 <unknown>\n#7 0x55bfa658e203 <unknown>\n#8 0x55bfa655ccc0 <unknown>\n#9 0x55bfa655dc9e <unknown>\n#10 0x55bfa69d2d0b <unknown>\n#11 0x55bfa69d6c92 <unknown>\n#12 0x55bfa69bfb3c <unknown>\n#13 0x55bfa69d7807 <unknown>\n#14 0x55bfa69a50df <unknown>\n#15 0x55bfa69f4578 <unknown>\n#16 0x55bfa69f4740 <unknown>\n#17 0x55bfa6a04196 <unknown>\n#18 0x7f08da294ac3 <unknown>","statusTrace":"self = <tests.forgot_password_test.TestForgotPassword object at 0x7f4c17588320>\n\n    @allure.description(\"Exception catching\")\n    @allure.title(\"Exception test\")\n    @allure.link(\"github.com/allure-examples/\", name=\"Allure Examples\")\n    @allure.issue(\"github.com/allure-examples/allure-examples/issues/1\", name=\"ISSUE-1\")\n    @allure.testcase(\n        \"github.com/allure-examples/allure-examples/issues/2\", name=\"TESTCASE-2\"\n    )\n    def test_expected_exception_on_page_title(self):\n>       self.about_page.click_login_link()\n\ntests/forgot_password_test.py:53: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \npages/about_page.py:20: in click_login_link\n    self.click(self.LOGIN_LINK)\npages/base_page.py:60: in click\n    el: WebElement = self.wait.until(\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <selenium.webdriver.support.wait.WebDriverWait (session=\"eecb034ca1347f902ddb7802f2ca94d6\")>\nmethod = <function element_to_be_clickable.<locals>._predicate at 0x7f4c17353ce0>\nmessage = ''\n\n    def until(self, method: Callable[[D], Union[Literal[False], T]], message: str = \"\") -> T:\n        \"\"\"Calls the method provided with the driver as an argument until the \\\n        return value does not evaluate to ``False``.\n    \n        :param method: callable(WebDriver)\n        :param message: optional message for :exc:`TimeoutException`\n        :returns: the result of the last call to `method`\n        :raises: :exc:`selenium.common.exceptions.TimeoutException` if timeout occurs\n        \"\"\"\n        screen = None\n        stacktrace = None\n    \n        end_time = time.monotonic() + self._timeout\n        while True:\n            try:\n                value = method(self._driver)\n                if value:\n                    return value\n            except self._ignored_exceptions as exc:\n                screen = getattr(exc, \"screen\", None)\n                stacktrace = getattr(exc, \"stacktrace\", None)\n            if time.monotonic() > end_time:\n                break\n            time.sleep(self._poll)\n>       raise TimeoutException(message, screen, stacktrace)\nE       selenium.common.exceptions.TimeoutException: Message: \nE       Stacktrace:\nE       #0 0x55bfa6a0531a <unknown>\nE       #1 0x55bfa651b6e0 <unknown>\nE       #2 0x55bfa656a3e6 <unknown>\nE       #3 0x55bfa656a681 <unknown>\nE       #4 0x55bfa65afb04 <unknown>\nE       #5 0x55bfa658e48d <unknown>\nE       #6 0x55bfa65aced7 <unknown>\nE       #7 0x55bfa658e203 <unknown>\nE       #8 0x55bfa655ccc0 <unknown>\nE       #9 0x55bfa655dc9e <unknown>\nE       #10 0x55bfa69d2d0b <unknown>\nE       #11 0x55bfa69d6c92 <unknown>\nE       #12 0x55bfa69bfb3c <unknown>\nE       #13 0x55bfa69d7807 <unknown>\nE       #14 0x55bfa69a50df <unknown>\nE       #15 0x55bfa69f4578 <unknown>\nE       #16 0x55bfa69f4740 <unknown>\nE       #17 0x55bfa6a04196 <unknown>\nE       #18 0x7f08da294ac3 <unknown>\n\n.venv/lib/python3.12/site-packages/selenium/webdriver/support/wait.py:105: TimeoutException","flaky":false,"newFailed":false,"newBroken":false,"newPassed":false,"retriesCount":0,"retriesStatusChange":false,"beforeStages":[{"name":"base_url","time":{"start":1733708980489,"stop":1733708980489,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"_verify_url","time":{"start":1733708980489,"stop":1733708980489,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"session_request","time":{"start":1733708980489,"stop":1733708980490,"duration":1},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false}],"testStage":{"description":"Exception catching","status":"broken","statusMessage":"selenium.common.exceptions.TimeoutException: Message: \nStacktrace:\n#0 0x55bfa6a0531a <unknown>\n#1 0x55bfa651b6e0 <unknown>\n#2 0x55bfa656a3e6 <unknown>\n#3 0x55bfa656a681 <unknown>\n#4 0x55bfa65afb04 <unknown>\n#5 0x55bfa658e48d <unknown>\n#6 0x55bfa65aced7 <unknown>\n#7 0x55bfa658e203 <unknown>\n#8 0x55bfa655ccc0 <unknown>\n#9 0x55bfa655dc9e <unknown>\n#10 0x55bfa69d2d0b <unknown>\n#11 0x55bfa69d6c92 <unknown>\n#12 0x55bfa69bfb3c <unknown>\n#13 0x55bfa69d7807 <unknown>\n#14 0x55bfa69a50df <unknown>\n#15 0x55bfa69f4578 <unknown>\n#16 0x55bfa69f4740 <unknown>\n#17 0x55bfa6a04196 <unknown>\n#18 0x7f08da294ac3 <unknown>","statusTrace":"self = <tests.forgot_password_test.TestForgotPassword object at 0x7f4c17588320>\n\n    @allure.description(\"Exception catching\")\n    @allure.title(\"Exception test\")\n    @allure.link(\"github.com/allure-examples/\", name=\"Allure Examples\")\n    @allure.issue(\"github.com/allure-examples/allure-examples/issues/1\", name=\"ISSUE-1\")\n    @allure.testcase(\n        \"github.com/allure-examples/allure-examples/issues/2\", name=\"TESTCASE-2\"\n    )\n    def test_expected_exception_on_page_title(self):\n>       self.about_page.click_login_link()\n\ntests/forgot_password_test.py:53: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \npages/about_page.py:20: in click_login_link\n    self.click(self.LOGIN_LINK)\npages/base_page.py:60: in click\n    el: WebElement = self.wait.until(\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <selenium.webdriver.support.wait.WebDriverWait (session=\"eecb034ca1347f902ddb7802f2ca94d6\")>\nmethod = <function element_to_be_clickable.<locals>._predicate at 0x7f4c17353ce0>\nmessage = ''\n\n    def until(self, method: Callable[[D], Union[Literal[False], T]], message: str = \"\") -> T:\n        \"\"\"Calls the method provided with the driver as an argument until the \\\n        return value does not evaluate to ``False``.\n    \n        :param method: callable(WebDriver)\n        :param message: optional message for :exc:`TimeoutException`\n        :returns: the result of the last call to `method`\n        :raises: :exc:`selenium.common.exceptions.TimeoutException` if timeout occurs\n        \"\"\"\n        screen = None\n        stacktrace = None\n    \n        end_time = time.monotonic() + self._timeout\n        while True:\n            try:\n                value = method(self._driver)\n                if value:\n                    return value\n            except self._ignored_exceptions as exc:\n                screen = getattr(exc, \"screen\", None)\n                stacktrace = getattr(exc, \"stacktrace\", None)\n            if time.monotonic() > end_time:\n                break\n            time.sleep(self._poll)\n>       raise TimeoutException(message, screen, stacktrace)\nE       selenium.common.exceptions.TimeoutException: Message: \nE       Stacktrace:\nE       #0 0x55bfa6a0531a <unknown>\nE       #1 0x55bfa651b6e0 <unknown>\nE       #2 0x55bfa656a3e6 <unknown>\nE       #3 0x55bfa656a681 <unknown>\nE       #4 0x55bfa65afb04 <unknown>\nE       #5 0x55bfa658e48d <unknown>\nE       #6 0x55bfa65aced7 <unknown>\nE       #7 0x55bfa658e203 <unknown>\nE       #8 0x55bfa655ccc0 <unknown>\nE       #9 0x55bfa655dc9e <unknown>\nE       #10 0x55bfa69d2d0b <unknown>\nE       #11 0x55bfa69d6c92 <unknown>\nE       #12 0x55bfa69bfb3c <unknown>\nE       #13 0x55bfa69d7807 <unknown>\nE       #14 0x55bfa69a50df <unknown>\nE       #15 0x55bfa69f4578 <unknown>\nE       #16 0x55bfa69f4740 <unknown>\nE       #17 0x55bfa6a04196 <unknown>\nE       #18 0x7f08da294ac3 <unknown>\n\n.venv/lib/python3.12/site-packages/selenium/webdriver/support/wait.py:105: TimeoutException","steps":[{"name":"Click Login link","time":{"start":1733708996672,"stop":1733709006933,"duration":10261},"status":"broken","statusMessage":"selenium.common.exceptions.TimeoutException: Message: \nStacktrace:\n#0 0x55bfa6a0531a <unknown>\n#1 0x55bfa651b6e0 <unknown>\n#2 0x55bfa656a3e6 <unknown>\n#3 0x55bfa656a681 <unknown>\n#4 0x55bfa65afb04 <unknown>\n#5 0x55bfa658e48d <unknown>\n#6 0x55bfa65aced7 <unknown>\n#7 0x55bfa658e203 <unknown>\n#8 0x55bfa655ccc0 <unknown>\n#9 0x55bfa655dc9e <unknown>\n#10 0x55bfa69d2d0b <unknown>\n#11 0x55bfa69d6c92 <unknown>\n#12 0x55bfa69bfb3c <unknown>\n#13 0x55bfa69d7807 <unknown>\n#14 0x55bfa69a50df <unknown>\n#15 0x55bfa69f4578 <unknown>\n#16 0x55bfa69f4740 <unknown>\n#17 0x55bfa6a04196 <unknown>\n#18 0x7f08da294ac3 <unknown>\n\n","statusTrace":"  File \"/home/runner/work/Selenium-Python-Example/Selenium-Python-Example/.venv/lib/python3.12/site-packages/allure_commons/_allure.py\", line 192, in impl\n    return func(*a, **kw)\n           ^^^^^^^^^^^^^^\n  File \"/home/runner/work/Selenium-Python-Example/Selenium-Python-Example/pages/about_page.py\", line 20, in click_login_link\n    self.click(self.LOGIN_LINK)\n  File \"/home/runner/work/Selenium-Python-Example/Selenium-Python-Example/pages/base_page.py\", line 60, in click\n    el: WebElement = self.wait.until(\n                     ^^^^^^^^^^^^^^^^\n  File \"/home/runner/work/Selenium-Python-Example/Selenium-Python-Example/.venv/lib/python3.12/site-packages/selenium/webdriver/support/wait.py\", line 105, in until\n    raise TimeoutException(message, screen, stacktrace)\n","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":true,"stepsCount":0,"attachmentsCount":0,"hasContent":true,"attachmentStep":false},{"name":"public ip address","time":{"start":1733709006954,"stop":1733709007095,"duration":141},"status":"passed","steps":[],"attachments":[{"uid":"c215806daa04a2a9","name":"HTTP logs of http://checkip.amazonaws.com/","source":"c215806daa04a2a9.txt","type":"text/plain","size":356}],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":1,"hasContent":true,"attachmentStep":false}],"attachments":[{"uid":"9b62c2070530f0c5","name":"Cookies","source":"9b62c2070530f0c5.json","type":"application/json","size":3586},{"uid":"fc915f297980c85c","name":"Session Storage","source":"fc915f297980c85c.json","type":"application/json","size":359},{"uid":"4344cd26d6124391","name":"Local Storage","source":"4344cd26d6124391.json","type":"application/json","size":791},{"uid":"256118bf6caed373","name":"Console Logs","source":"256118bf6caed373.json","type":"application/json","size":236},{"uid":"bc148fd9ff15aac9","name":"Full Page Screenshot","source":"bc148fd9ff15aac9.png","type":"image/png","size":4170230},{"uid":"a8b1e1597d241f92","name":"URL","source":"a8b1e1597d241f92.uri","type":"text/uri-list","size":23},{"uid":"4b33c2d14d6d47ec","name":"Network Logs","source":"4b33c2d14d6d47ec.json","type":"application/json","size":4214807}],"parameters":[],"shouldDisplayMessage":true,"stepsCount":2,"attachmentsCount":8,"hasContent":true,"attachmentStep":false},"afterStages":[{"name":"session_request::0","time":{"start":1733709057390,"stop":1733709057390,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false}],"labels":[{"name":"severity","value":"critical"},{"name":"suite","value":"Custom suite"},{"name":"parentSuite","value":"Custom parent suite"},{"name":"subSuite","value":"Custom sub suite"},{"name":"story","value":"Forgot Password Feature's Functionality"},{"name":"epic","value":"Security"},{"name":"tag","value":"security"},{"name":"host","value":"fv-az2022-866"},{"name":"thread","value":"1785-MainThread"},{"name":"framework","value":"pytest"},{"name":"language","value":"cpython3"},{"name":"package","value":"tests.forgot_password_test"},{"name":"resultFormat","value":"allure2"}],"parameters":[],"links":[{"name":"TESTCASE-2","url":"https://github.com/allure-examples/allure-examples/issues/2","type":"tms"},{"name":"ISSUE-1","url":"https://github.com/allure-examples/allure-examples/issues/1","type":"issue"},{"name":"Allure Examples","url":"https://github.com/allure-examples/","type":"link"}],"hidden":false,"retry":false,"extra":{"severity":"critical","retries":[],"categories":[{"name":"Test defects","matchedStatuses":[],"flaky":false}],"tags":["security"]},"source":"7dbcdd63594319d0.json","parameterValues":[]}